app/javascript/packs/app/application.scss

```diff
@@ -71,4 +71,18 @@ pre:not(.ruby) {
   }
 }
 
+.light-switch:before {
+  content: '☾';
```

Comment:
I've been using https://fontawesome.com for the UI icons, which you can find [here](https://github.com/rubyapi/rubyapi/blob/master/app/javascript/packs/app/icons.js). Would we be able to use the icons here as well?

Reply:
Let‘s do this in later Pull Request? I did not find ones on fontawesome that are better than current unicode ones. Ideally we could have CSS icons like the ones in https://justjavascript.com/.

Reply:
> I did not find ones on fontawesome that are better than current unicode ones.

Moon: https://fontawesome.com/icons/moon?style=solid

Sun: https://fontawesome.com/icons/sun?style=solid (a bit another, but seems OK)

Bulb icons (like in Twitter) are paid in Font Awesome.

Also sometimes I think we can have too many icons/buttons in the header, and there will be some drop-down menu, and then it'd be better to do just toggle with text, without icon.